libc/src/math/generic/scalbnf128.cpp

```diff
@@ -14,9 +14,7 @@
 namespace LIBC_NAMESPACE {
 
 LLVM_LIBC_FUNCTION(float128, scalbnf128, (float128 x, int n)) {
-#if !defined(FLT_RADIX)
-#error FLT_RADIX undefined.
-#elif FLT_RADIX != 2
+#if FLT_RADIX != 2
```

Comment:
Do we need a `#else` here? I suspect not.  This could simply be:
```c
#if FLT_RADIX != 2
#error ...
#endif
  return fputil::...
```

or even moved outside of the function definition to make the definition crystal clear (1 statement).

Reply:
Done.